app/views/layouts/_base.html.erb

```diff
@@ -11,6 +11,7 @@
   <meta name="x-section-name" content="Business" />
   <meta name="x-section-link" content="/browse/business" />
   <meta name="x-section-format" content="trade-tariff" />
+  <meta name="description" content="<%= I18n.t('meta_description') %>">
```

Comment:
@jackscotti https://github.com/alphagov/trade-tariff-frontend/blob/db0d0d789d7bd540db90529156e8e247ebb871e6/app/controllers/application_controller.rb#L80 location of the artifact